src/Missions.java

```diff
@@ -3,24 +3,92 @@
 public class Missions {
     public static Scanner scanner = new Scanner(System.in);
 
-    public static int inputNumber(){
-        System.out.println("Enter a number:");
-        return scanner.nextInt();
+    public static void infinitePrintMax(int... numbers) {
+        System.out.println(Util.max(numbers));
     }
 
-    public static int maxNumber(){
-        int number = inputNumber();
+    public static void printAverage(double number1, double number2, double number3) {
+        System.out.println(Util.average(number1, number2, number3));
+    }
+
+    public static void printIsBothEven(double number1, double number2) {
+        if (Util.isEven(number1) && Util.isEven(number2)) {
+            System.out.println("Both numbers are even");
+        } else {
+            System.out.println("Womp Womp not both of the numbers are even");
+        }
+    }
+
+    public static void isBiggerThenAverage(double number1, double number2) {
+        if (Util.average(number1, number2) > number1) {
+            System.out.println("No way!!! the average is bigger then Herman's bana-sorry i meant the first number");
+        } else {
+            System.out.println("The average is smaller then the second number");
+        }
+    }
+
+    public static void maxBankHandle() {
+        String message = "Enter the amount of money the bank has:";
+        int bank1 = Util.inputInt(message);
+        int bank2 = Util.inputInt(message);
+        int bank3 = Util.inputInt(message);
+        infinitePrintMax(bank1, bank2, bank3);
+    }
+
+    public static void printIsEvenLength(int number) {
```

Comment:
if its exercise call it like that, u can call a function ifEvenPrintLength() but i think its ugly and name that is breaking SRP